unit_tests/test_nemesis.py

```diff
@@ -151,3 +153,21 @@ def test_list_topology_changes_monkey():
     for disrupt_method in collected_disrupt_methods_names:
         assert disrupt_method in expected_disrupt_method_names, \
             f"{disrupt_method=} from {collected_disrupt_methods_names=} was not found in {expected_disrupt_method_names=}"
+
+
+def test_disabled_monkey():
+
+    ToggleGcModeMonkey.disabled = True
+
+    tester = FakeTester()
+
+    all_disrupt_methods = {attr[1].__name__ for attr in inspect.getmembers(Nemesis) if
+                           attr[0].startswith('disrupt_') and
+                           callable(attr[1])}
+
+    tester.params["nemesis_include_filter"] = ['!disabled']
```

Comment:
This `include` is actually doing "exclude", IIUC. sounds confusing.
1. why not doing something like setting a new parameter of `nemesis_exclude_filter` with a default of `disabled` ?
2. Or else,  shouldn't `nemesis_include_filter`  have a default value of `!disabled` ?

Reply:
Quite agree with @yarongilor.
Is "disabled" property for nemesis a new thing?

Reply:
> feature(SisyphusMonkey): negative flag for `nemesis_include_filter`

Reply:
`disabled` is just an example. if we decide that we do introduce such a flag to the nemesis, yes it can be part of the default filter

I would rather have one configuration value, otherwise you need merge both, and handle conflicts (if any) 

as for the name `nemesis_include_filter`, 
I do think it should be called `nemesis_selector` or something like that, since `include` mean we want to add something, `filter` means when want to remove something. (a comment I've raised in the the PR started the whole discussion)

Reply:
I've renamed the configuration value to `nemesis_selector`